topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -169,12 +173,14 @@ On the x-axis are the base position in the read, and on the y-axis quality score
 
 > ### {% icon question %} Questions
 >
-> 1. How is the mean score changing along the sequence?
-> 2. Is this tendency represent all sequences?
+> 1. Which Phred encoding is used in the FASTQ file for these sequences?
+> 2. How is the mean score changing along the sequence?
+> 3. Is this tendency represent all sequences?
```

Comment:
For "Is this tendency represent all sequences?" maybe instead "Is this tendency seen in all sequences?"?